### ISSUE TITLE
Updating to 0.18.6 removed presets

### ISSUE THREAD

**Issue by verywetfire** [chaiNNer-org/chaiNNer#1757]
- Chainner version: 0.18.6
- OS: Windows 10

**Description**
Run the program. Decided to update, click update button, go to website, download and run updater. Afterwards, update removed all presets (.chn). Program installed on C:\ presets were on G:\  so not even sure how that's possible, but it happened.

**Logs**

[main.zip](https://github.com/chaiNNer-org/chaiNNer/files/11299993/main.zip)
[renderer.zip](https://github.com/chaiNNer-org/chaiNNer/files/11299994/renderer.zip)

**Comment by joeyballentine** [comment]
This makes no sense. Someone else just reported the same thing, but they were saving their chn files in the app's directory, which gets removed when it updates, so it was at least explainable. The fact that yours got removed on a different drive is concerning and I'm not even sure how that's possible

**Comment by joeyballentine** [comment]
hmmm taking a look at your renderer.log, it seems the pagefile.sys file on your G drive got locked up. I'm not really sure what it was trying to do at that point, but that could have something to do with it

**Comment by verywetfire** [comment]
> hmmm taking a look at your renderer.log, it seems the pagefile.sys file on your G drive got locked up. I'm not really sure what it was trying to do at that point, but that could have something to do with it

Yeah that was the oddest part. If it were same directory/drive, etc. ok, I could at least understand that something perhaps was connected but this was just random.

I'll just be sure to make extra copy of presets "in case of" before updating next time.

**Comment by joeyballentine** [comment]
I don't think this was an issue with chaiNNer, so I'm closing the issue
